.github/workflows/diff_report.yml

```diff
@@ -176,13 +176,16 @@ jobs:
          BASE_BRANCH="upstream/master"
          if [ -f new_module_config.xml ]; then
            groovy diff.groovy -r $REPO -p $REF -pc new_module_config.xml -m single\
-             -l project.properties -xm "-Dcheckstyle.failsOnError=false"
+             -l project.properties -xm "-Dcheckstyle.failsOnError=false"\
+             --allowExcludes
```

Comment:
> I thought we were only talking about the OpenJDK run, not for all report generation.

This is my only concern is that we are enabling this for all projects. We should re-look at our exlcude list and see if there is anything we want to remove.

Reply:
@romani Please confirm you are ok with this.

Reply:
it took me awhile to meditate on this...

having no excludes were ok for us to see easily that parsing is of files changed and here is diff. If somebody changing ANTRL it become useful. We do not change ANTLR that often. So probably should keep this in mind by maintainers, and let all other 95% of updates in code base to reuse less confusing environment for execution of openjdk sources.

I am ok to allowExcludes, I think it will make process more easy for outside contributors.
Excluded files because they are bad in complication is good. If we exclude some files due to new syntax, we should keep link "until .... " and our link check will catch forgotten exclude. But still it happens so rare.